packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -256,6 +299,7 @@ try {
 	apiEndpoint.__internal_setRequestHandler(requestHandler);
 
 	const primaryPhp = await requestHandler.getPrimaryPhp();
+	downloadWordPressAssets(primaryPhp);
```

Comment:
What do you think of the possibility of this download competing for bandwidth while the browser is initially requesting static assets? Could this adversely affect initial page load, and if so, would it be better to start this download later, possibly after the initial WP page finishes loading?

Reply:
I don't think that it's a big deal. 
I tested this locally with preview, the only files that loaded after the static zip were files from that zip.

After the first load, the zip will be cached so it won't take up bandwidth.

If it turns out to be a problem we could move it to window.onload in the website.

Reply:
In [b2ba22b](https://github.com/WordPress/wordpress-playground/pull/1532/commits/b2ba22b64db60dc8d495cea3123771979e3258f1) the download will be started once remote is loaded.